dockerfiles/proxy/access_by_openidc.lua

```diff
@@ -0,0 +1,102 @@
+local cjson = require("cjson")
+
+-- configs
+local opts = {
+    claim_groups = ngx.var.oidc_claim_groups or "https://sso.mozilla.com/claim/groups",
+    client_id = ngx.var.oidc_client_id,
+    discovery = ngx.var.oidc_discovery or "https://auth.mozilla.auth0.com/.well-known/openid-configuration",
+    logout_path = ngx.var.oidc_logout_path,
+    redirect_uri = ngx.var.oidc_redirect_uri,
+    scope = ngx.var.oidc_scope or "openid email profile",
```

Comment:
For the future I'm thinking `OIDC_SCOPE="email"` would be all we need to associate a Django user to an authenticated user.

Do you know what `profile` sends?

Reply:
This is more or less copied from what we use for other applications so I just left it as is. `openid` is required for doing the token exchange, and `profile` is used to get the users LDAP groups, which may not be needed for us at the moment but may be useful in the future.

I may need to potentially add the `read:aggregates` scope since we specified that when we set up the Auth0 creds initially.

Reply:
I think we can leave scopes out for now. If we need scopes in the future we can add something in.